add: Fork concept under git category (#3705)
Codecademy/docs: content/git/concepts/fork/fork.md

---
Title: 'Fork'
Description: 'Making a duplicate of a repository to work on separately is known as forking.'
Subjects: 
  - 'Code Foundations'
  - 'Developer Tools'
  - 'Web Development'
Tags:
  - 'Git'
  - 'GitHub'
  - 'Version Control'
CatalogContent:
  - 'learn-git'
  - 'learn-the-command-line'
---

> **Fork** is making a duplicate of an existing repository. This "fork" becomes a separate undertaking that you are in charge of. Collaboration, experimentation, and contributions to open-source code are made possible by forking. It enables developers worldwide to collaborate, make changes, and contribute to projects while retaining the integrity of the source code. It is a secure approach to modify and test code without changing the original.

Reviewer: ```suggestion
A **fork** is making a duplicate of an existing repository. This "fork" becomes a separate undertaking that the user is in charge of. Collaboration, experimentation, and contributions to open-source code are made possible by forking. It enables developers worldwide to collaborate, make changes, and contribute to projects while retaining the integrity of the source code. It is a secure approach to modify and test code without changing the original.
```
We avoid first- and second-person pronouns in docs.

## Steps to Fork a Repository

> ### Step 1

Reviewer: Instead of headers, use a bulleted list, and remove the second-person pronouns ("you")

First, Login to your GitHub account.

### Step 2

Navigate to the Github repository you want to fork.

### Step 3

Click the "Fork" button in the top right corner.

> ![Repo page with Fork button](https://i.imgur.com/000HPJD.png)

Reviewer: If you want to put an image here, follow the instructions on hosting media in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting).

> This will create a new copy of the original "Codecademy/docs" repository into your account on which you can work.

Reviewer: ```suggestion
This will create a new copy of the original "Codecademy/docs" repository.
```

---
Title: 'Fork'
Description: 'Making a duplicate of a repository to work on separately is known as forking.'
Subjects: 
  - 'Code Foundations'
  - 'Developer Tools'
  - 'Web Development'
Tags:
  - 'Git'
  - 'GitHub'
  - 'Version Control'
CatalogContent:
  - 'learn-git'
---

A **fork** is making a duplicate of an existing repository. This "fork" becomes a separate undertaking that the user is in charge of. Collaboration, experimentation, and contributions to open-source code are made possible by forking. It enables developers worldwide to collaborate, make changes, and contribute to projects while retaining the integrity of the source code. It is a secure approach to modify and test code without changing the original.

> ## Steps to Fork a Repository

Reviewer: You should note that the fork command is provided by a separate command line tool than Git.

+ Step 1

> First the user login to his/her GitHub account using the command

Reviewer: ```suggestion
First the user must login to his/her GitHub account using the command:
```

```shell
gh auth login
```

+ Step 2

> After successful login, the user can fork the repository using the command

Reviewer: ```suggestion
After successful login, the user can fork the repository using the command:
```

---
Title: 'Fork'

> Description: 'Making a duplicate of a repository to work on separately is known as forking.'

Reviewer 2: ```suggestion
Description: 'A duplicate of a repository that resides within the independent directories of a contributor.'
```

Subjects: 
  - 'Code Foundations'
  - 'Developer Tools'

>   - 'Web Development'

Reviewer 2: ```suggestion
```

Reviewer 2: subjects should only contain 2 items

Tags:
  - 'Git'
  - 'GitHub'
  - 'Version Control'
CatalogContent:

>   - 'learn-git'

Reviewer 2: ```suggestion
  - 'learn-git'
  - 'learn-the-command-line'
```

---

> A **fork** is making a duplicate of an existing repository. This "fork" becomes a separate undertaking that the user is in charge of. Collaboration, experimentation, and contributions to open-source code are made possible by forking. It enables developers worldwide to collaborate, make changes, and contribute to projects while retaining the integrity of the source code. It is a secure approach to modify and test code without changing the original.

Reviewer 2: ```suggestion
A **fork** is a duplicate of an existing repository. This "fork" is an independent entity that allows a contributor to interact with the original repository. Collaboration, experimentation, and contributions to open-source code are made possible by forking. It enables developers worldwide to collaborate, make changes, and contribute to projects while retaining the integrity of the source code. It is a secure approach to modify and test code without making changes directly to the original.
```

> **Note:** The fork command is provided by a separate command-line tool than Git.

## Steps to Fork a Repository

> 

Reviewer 2: It may be helpful for those less familiar with Git/version control to mention where to acquire a link for a given repo
